package.json

```diff
@@ -51,6 +54,7 @@
     "load-grunt-tasks": "5.1.0",
     "nock": "^13.3.8",
     "octokit": "^3.1.2",
+    "prettier": "^3.1.1",
     "style-loader": "^3.3.3",
     "supertest": "^6.3.3",
     "typescript": "^5.3.2",
```

Comment:
`typescript` Is this necessary?

Reply:
you can run `docker compose run website yarn npm-check` to check it, but i think this was needed for some webpack commands. not sure though

Reply:
just checked it, it is used by the `node-dependency-injection`, they used it to autowire services and i never had a chance to investigate on how to disable it.

Reply:
```
website-website-1  |     Error: Cannot find module 'typescript'
website-website-1  |     Require stack:
website-website-1  |     - /code/node_modules/@typescript-eslint/typescript-estree/dist/convert.js
website-website-1  |     - /code/node_modules/@typescript-eslint/typescript-estree/dist/ast-converter.js
website-website-1  |     - /code/node_modules/@typescript-eslint/typescript-estree/dist/parser.js
website-website-1  |     - /code/node_modules/@typescript-eslint/typescript-estree/dist/index.js
website-website-1  |     - /code/node_modules/node-dependency-injection/dist/lib/Autowire.js
website-website-1  |     - /code/node_modules/node-dependency-injection/dist/lib/Loader/FileLoader.js
website-website-1  |     - /code/node_modules/node-dependency-injection/dist/lib/Loader/JsFileLoader.js
website-website-1  |     - /code/node_modules/node-dependency-injection/dist/lib/index.js
website-website-1  |     - /code/src/backend/dependency-injection/AppContainer.js
website-website-1  |     - /code/src/backend/AppKernel.js
website-website-1  |     - /code/src/backend/index.js

```